packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -328,6 +329,7 @@ contract Deploy is Deployer {
 
         setAlphabetFaultGameImplementation({ _allowUpgrade: false });
         setCannonFaultGameImplementation({ _allowUpgrade: false });
+        setPermissionedCannonFaultGameImplementation({ _allowUpgrade: false });
```

Comment:
This will always create the permissioned game but just not set it as the respectedGameType unless explicitly configured to do so, correct?

Reply:
Yes, by default we roll w/ `CANNON`